src/main/java/beam/physsim/cch/RoutingFrameworkWrapper.scala

```diff
@@ -0,0 +1,211 @@
+                                         |docker run --rm
+                                         | -v $tempDir:/work
+                                         | $toolDockerImage
+                                         | $createODPairsLauncher
+                                         | -g $graphPathInContainer
+                                         | -n 1000
+                                         | -o ${odPairsFileInContainer(0, 0)} -d 10 15 20 25 30 -geom
+      """.stripMargin.replace("\n", ""))
+
+    createODPairsOutput.lineStream.foreach(logger.info(_))
+  }
+
+  def writeOds(iteration: Int, hour: Int, ods: Stream[OD]): Unit = {
+    itHourRelatedPath(tempDirPath, iteration, hour, "").toFile.mkdirs()
+    val odPairsFile = odPairsFileInTempDir(iteration, hour).toFile
+
+    val writer = new BufferedWriter(new FileWriter(odPairsFile))
+    writer.write("origin,destination")
+    writer.newLine()
+
+    ods.foreach {
+      case OD(first, second) =>
+        writer.write(s"$first,$second")
+        writer.newLine()
+    }
+
+    writer.close()
+  }
+
+  override def assignTrafficAndFetchWay2TravelTime(iteration: Int, hour: Int): Map[Long, Double] = {
+    val flowPath = itHourRelatedPath("/work", iteration, hour, "flow")
+    val distPath = itHourRelatedPath("/work", iteration, hour, "dist")
+    val statPath = itHourRelatedPath("/work", iteration, hour, "stat")
+
+    val query = s"""
+                     |docker run --rm
+                     | --memory-swap -1
+                     | -v $tempDir:/work
+                     | $toolDockerImage
+                     | $assignTrafficLauncher
+                     | -g $graphPathInContainer
+                     | -d ${odPairsFileInContainer(iteration, hour)}
+                     | -p 1 
+                     | -n 0 
+                     | -o random
+                     | -i
+                     | ${if (verboseLoggingEnabled) "-v" else ""}
+                     | -flow $flowPath
+                     | -dist $distPath
+                     | -stat $statPath
+      """.stripMargin.replace("\n", "")
+
+    if (verboseLoggingEnabled)
+      logger.info("Docker command for assigning traffic: {}", query)
+
+    val assignTrafficOutput = Process(query)
+
+    assignTrafficOutput.lineStream.foreach(logger.info(_))
+
+    var curIter = -1
+    val wayId2TravelTime = new mutable.HashMap[Long, Double]()
+
+    Source
+      .fromFile(itHourRelatedPath(tempDirPath, iteration, hour, "flow.csv").toFile)
+      .use { source =>
+        // drop headers
+        source
+          .getLines()
+          .drop(2)
+          .map(x => x.split(","))
+          // picking only result of last iteration
+          .map { x =>
+            if (x(0).toInt != curIter) {
+              curIter = x(0).toInt
+              wayId2TravelTime.clear()
+            }
+            x
+          }
+          // way id into BPR'ed travel time
+          .map(x => x(4).toLong -> x(5).toDouble / 10.0)
```

Comment:
extract column name variables and introduce variable for 10

Reply:
done